### PR TITLE
procs: init at 0.8.5

### PR DESCRIPTION
**Pull request by dalance**
###### Motivation for this change

This PR adds procs, A modern replacement for ps written by Rust.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/admin/procs/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    sha256 = "0ihww2sm9hnh748723lr1cxw9zyi9nfxbbiij5a465mypa2p7w0v";
+  };
+
+  cargoSha256 = "1aq2nhspb9kp9mzj5550xph09qvd0ahlw246hcx2mqkr4frh64x0";
+
+  buildInputs = stdenv.lib.optional stdenv.isDarwin Security;
+
+  meta = with stdenv.lib; {
+    description = "A modern replacement for ps written by Rust";
+    homepage = https://github.com/dalance/procs;
```
Comment (JohnAZoidberg) [inline pkgs/tools/admin/procs/default.nix]: Hint: We're probably going to retire unquoted URL syntax when RFC 45 is accepted.

File: pkgs/tools/admin/procs/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    rev = "v${version}";
+    sha256 = "0ihww2sm9hnh748723lr1cxw9zyi9nfxbbiij5a465mypa2p7w0v";
+  };
+
+  cargoSha256 = "1aq2nhspb9kp9mzj5550xph09qvd0ahlw246hcx2mqkr4frh64x0";
+
+  buildInputs = stdenv.lib.optional stdenv.isDarwin Security;
+
+  meta = with stdenv.lib; {
+    description = "A modern replacement for ps written by Rust";
```
Comment (JohnAZoidberg) [inline pkgs/tools/admin/procs/default.nix]:
```suggestion
    description = "A modern replacement for ps written in Rust";
```

**Comment by JohnAZoidberg** [comment]
Please split it up into two commits and name them as the manual requires:

```
maintainers: add dalance
procs: init at 0.8.5
```

**Comment by c0bw3b** [comment]
@GrahamcOfBorg build procs

**Review by c0bw3b — APPROVED** [review]
Tested locally and it works :heavy_check_mark:

**Comment by c0bw3b** [comment]
Thanks for the contribution to nixpkgs @dalance and great tool :)

**Comment by dalance** [comment]
Thanks!

**Pull request merged**